src/components/sidebar/WatchlistAddButton/index.tsx

```diff
@@ -58,7 +58,7 @@ const WatchlistAddButton = () => {
             sx={{ py: 1.3 }}
             startIcon={<VisibilityOutlined sx={{ verticalAlign: 'middle', marginRight: 1 }} />}
           >
-            Add read-only
+            Add as Read-only
```

Comment:
```suggestion
            Add as read-only
```

We don't capitalize common words. If it's capitalized in the design, please ask @TanyaEfremova to lowercase it.